tools/backsub/backsub.xml

```diff
@@ -0,0 +1,96 @@
+<tool id="backsub" name="Background subtraction" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+
+    <description>for sequential immunofluorescence images.</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+ 
+    <expand macro="requirements"/>
+    <expand macro="version_cmd"/>
+
+    <command detect_errors="aggressive"><![CDATA[
+
+        @CMD_BEGIN@ 
```

Comment:
Imho this is a bit too much macros :) Its not helping for a review and does not reduce repetition as its only used one time.